### PR TITLE
Bumped cephfs packages to luminous

### PR DESCRIPTION
**Pull request by farcaller**
Updated ceph common packages to luminous in cephfs.

### CODE REVIEW

**Comment by farcaller** [comment]
Note: tests failed for this and I lack any centos/rpm expertise to figure what's wrong there. Any ideas?

**Comment by rootfs** [comment]
@farcaller instead of using pkg from ceph.com, try `yum install -y centos-release-ceph`

**Comment by farcaller** [comment]
Done.

File: ceph/cephfs/Dockerfile.release [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN go build -a -ldflags '-extldflags "-static"' -o /go/bin/cephfs-provisioner c
 
 FROM centos:7
 
-ENV CEPH_VERSION "jewel"
+ENV CEPH_VERSION "luminous"
 RUN rpm -Uvh https://download.ceph.com/rpm-$CEPH_VERSION/el7/noarch/ceph-release-1-1.el7.noarch.rpm && \
   yum install -y epel-release && \
   yum install -y ceph-common python-cephfs && \
```
[the remark this answers is not in the document]
Comment (farcaller) [reply ceph/cephfs/Dockerfile.release]: Done.

**Pull request merged**